docker-compose/docker-compose.yml

```diff
@@ -76,12 +78,19 @@ services:
             - 8099:8099
 
     share:
-        image: alfresco/alfresco-share:6.0
+        image: alfresco-share-test
```

Comment:
Looks like we are going to have to wait for a Share release.
Might be better to not change the Share version and to get the rest of the changes merged to master so they will be picked up when there is a Share release.

Reply:
This one slipped into the commit, I`m going to revert this change.